src/main/java/com/conveyal/datatools/manager/models/Deployment.java

```diff
@@ -185,7 +185,7 @@ public Date retrieveLastDeployed () {
     }
 
     /**
-     * The routerId of this deployment
+     * The routerId of this deployment. If null, this value defaults to "default".
```

Comment:
It looks like the comment means more: "If null, the deployment will use/target the 'default' router."